### ISSUE TITLE
don't know why it caused

### ISSUE THREAD

**Issue by Sorata512** [mukul500/AnimeXStream#148]
First time-
The video is not playing and the search is not working normally and the dark theme is not showing though many mobile is in dark mode

But it's video is playing and search is working when I use it with VPN
After that it search and video both are working properly 
without VPN but the night mode is still not coming

**Comment by MfundoZA** [comment]
Yeah, I have the same issue with the buffer. Even opened an issue two months ago. 

Honestly I'd do it myself if I could but I still don't know even about mobile development.

**Comment by rafayumair** [comment]
Having the Same Problem

**Comment by rafayumair** [comment]
I fixed the Problem, Gogoanime is changing its url, So change the Url
from
AnimeXStream-0.2.4\app\src\main\java\net\xblacky\animexstream\utils\constants\C.kt
file
var BASE_URL = "https://gogoanime.so/"
const val ORIGIN = "https://gogoanime.so/"
        const val  REFERER = "https://gogoanime.so/"

**Comment by Melmis** [comment]
> I fixed the Problem, Gogoanime is changing its url, So change the Url
> from
> AnimeXStream-0.2.4\app\src\main\java\net\xblacky\animexstream\utils\constants\C.kt
> file
> var BASE_URL = "https://gogoanime.so/"
> const val ORIGIN = "https://gogoanime.so/"
>         const val  REFERER = "https://gogoanime.so/"

Did you do it? Or do we need to wait for the Dev to fix it

**Comment by rafayumair** [comment]
> Did you do it? Or do we need to wait for the Dev to fix it

Replace code in this file AnimeXStream-0.2.4\app\src\main\java\net\xblacky\animexstream\utils\constants\C.kt

to

`package com.charlie.animeing.utils.constants

class C {
    companion object{

        const val GIT_DOWNLOAD_URL = "https://github.com/mukul500/AnimeXStream/"

        //Error Codes
        const val RESPONSE_UNKNOWN: Int = 1000
        const val ERROR_CODE_DEFAULT: Int = -1000
        const val NO_INTERNET_CONNECTION = 1001

        //Base URLS
        var BASE_URL = "https://gogoanime.so/"
        const val EPISODE_LOAD_URL = "https://ajax.gogocdn.net/ajax/load-list-episode"
        const val SEARCH_URL = "/search.html"

        //Model Type
        const val TYPE_RECENT_SUB = 1
        const val TYPE_POPULAR_ANIME =2
        const val TYPE_RECENT_DUB = 3
        const val TYPE_GENRE = 4
        const val TYPE_MOVIE = 5
        const val TYPE_NEW_SEASON = 6
        const val TYPE_DEFAULT= -1

        // Retrofit Request TYPE

        const val RECENT_SUB = 1
        const val RECENT_DUB = 2

        const val MAX_LIMIT_FOR_SUB_DUB = 10


        const val NEWEST_SEASON_POSITION = 2
        const val RECENT_SUB_POSITION = 0
        const val RECENT_DUB_POSITION = 1
        const val POPULAR_POSITION = 4
        const val MOVIE_POSITION = 3

        //Episode URL Type
        const val TYPE_MEDIA_URL = 100
        const val TYPE_M3U8_URL = 101

        //Anime Info URL Type
        const val TYPE_ANIME_INFO = 1000
        const val TYPE_EPISODE_LIST = 1001
        const val M3U8_REGEX_PATTERN = "(http|https)://([\\w_-]+(?:(?:\\.[\\w_-]+)+))([\\w.,@?^=%&:/~+#-]*[\\w@?^=%&/~+#-])?"

        //Anime Search Types
        const val TYPE_SEARCH_NEW = 2000
        const val TYPE_SEARCH_UPDATE = 2001

        //Network Requests Header
        const val USER_AGENT = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36"
        const val ORIGIN = "https://gogoanime.so/"
        const val  REFERER = "https://gogoanime.so/"

        //Realm
        const val MAX_TIME_M3U8_URL = 2 * 60 * 60 *1000
        const val MAX_TIME_FOR_ANIME = 2 * 24 * 60 *60 * 1000
    }
}`

**Comment by danielr0206** [comment]
> > I fixed the Problem, Gogoanime is changing its url, So change the Url
> > from
> > AnimeXStream-0.2.4\app\src\main\java\net\xblacky\animexstream\utils\constants\C.kt
> > file
> > var BASE_URL = "https://gogoanime.so/"
> > const val ORIGIN = "https://gogoanime.so/"
> > const val  REFERER = "https://gogoanime.so/"
> 
> Did you do it? Or do we need to wait for the Dev to fix it

Hey so did you change the code? Is it working?... Just asking cuz I have some issues with the stream... If yes please tell what to do...
